build/build.xml

```diff
@@ -640,6 +646,7 @@
 				<include name="selenium-${status}-*.zap"/>
 				<include name="spiderAjax-${status}-*.zap"/>
 				<include name="websocket-${status}-*.zap"/>
+				<include name= "fieldenumeration-${status}-*.zap"/>
```

Comment:
This is going to be included in the release build?

Reply:
This change should probably done later, when more stable (we don't usually have alpha add-ons in the main release).